everware/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.9.0"
```

Comment:
why did you chose to introduce separate file instead of adding it to __init__.py?
(see example at https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/__init__.py)